src/client/wallet.go

```diff
@@ -184,3 +184,8 @@ func (r *WalletRequester) SignTx(message []byte) (*api.ApiResponse[api.WalletSig
 	}
 	return SendGetRequest[api.WalletSignTxData](r, "sign-tx", "SignTx", args)
 }
+
+func (r *WalletRequester) ClaimRewards() (*api.ApiResponse[api.SuccessData], error) {
```

Comment:
This needs to be put in `src/modules/stakewise/client/wallet`, not here; `/src/client/wallet` is for general Hyperdrive API functions but claiming is a Stakewise function.

EDIT: I see it's in there as well, so this entire block can just be removed.